staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -54,6 +54,15 @@ const (
 	// Allows us to enable anonymous auth for only certain apiserver endpoints.
 	AnonymousAuthConfigurableEndpoints featuregate.Feature = "AnonymousAuthConfigurableEndpoints"
 
+	// owner: @stlaz @tkashem @dgrisonnet
```

Comment:
sounded like standa wanted to be removed.

Reply:
no, he is fine to be here, we talked to him